climage/__main__.py

```diff
@@ -23,13 +31,85 @@ def _get_color_type(is_truecolor, is_256color, is_16color, is_8color):
     )
 
     if is_truecolor:
-        return _color_types.truecolor
+        return color_types.truecolor
     if is_256color:
-        return _color_types.color256
+        return color_types.color256
     if is_16color:
-        return _color_types.color16
+        return color_types.color16
     if is_8color:
-        return _color_types.color8
+        return color_types.color8
+
+
+def color_to_flags(color_type):
```

Comment:
- [ ] TODO: Can write this better.

```
def color_to_flags(color_type):
    """
    Convert a value of the color_types enum to kwargs accepted by the conversion functions.
    """
    ret = {
        "is_truecolor": color_type == color_types.truecolor,
        "is_256color": color_type == color_types.color256,
        "is_16color": color_type == color_types.color16,
        "is_8color": color_type == color_types.color8,
    }
    assert any(ret.values()), "Invalid color type supplied"
    return ret
```

Haven't tested that assert, but that's about what I want to do.